coderd/prometheusmetrics/prometheusmetrics_test.go

```diff
@@ -511,23 +511,35 @@ func TestAgentStats(t *testing.T) {
 func TestExperimentsMetric(t *testing.T) {
 	t.Parallel()
 
+	if len(codersdk.ExperimentsAll) == 0 {
+		t.Skip("No experiments are currently defined; skipping test.")
+	}
+
 	tests := []struct {
 		name        string
 		experiments codersdk.Experiments
 		expected    map[codersdk.Experiment]float64
 	}{
 		{
 			name:        "Enabled experiment is exported in metrics",
-			experiments: codersdk.Experiments{codersdk.ExperimentSharedPorts},
+			experiments: codersdk.Experiments{
+				// TODO: We need to use a real experiment from codersdk.ExperimentsAll
+				// when the slice is not empty. Uncomment when that happens.
+				// codersdk.ExperimentSharedPorts,
```

Comment:
why not just do `codersdk.ExperimentsAll[0]`?

Reply:
because I'm not as smart as you dean